services/core-web/src/components/Forms/incidents/incidentFormDocuments.tsx

```diff
@@ -0,0 +1,158 @@
+import { IMineIncidentDocument, INCIDENT_DOCUMENT_TYPES } from "@mds/common";
```

Comment:
If there are other things to fix, I would advocate for renaming incidentFormDocuments and incidentFormInitialReport to use PascalCase.